aptos-move/e2e-move-tests/src/tests/keyless_feature_gating.rs

```diff
@@ -86,6 +88,36 @@ fn test_feature_gating(
     }
 }
 
+#[test]
+fn naive_bench() {
+    let (mut h, recipient, core_resources) = init_feature_gating(
+        vec![
+            FeatureFlag::CRYPTOGRAPHY_ALGEBRA_NATIVES,
+            FeatureFlag::BN254_STRUCTURES,
+            FeatureFlag::KEYLESS_ACCOUNTS,
+        ],
+        vec![],
+    );
+
+    // Old proof for old VK
+    let (old_sig, pk) = get_sample_groth16_sig_and_pk();
+    let account = create_keyless_account(&mut h, pk);
+
+    let mut all_times = vec![];
+    for i in 0..99 {
+        let transaction =
+            spend_keyless_account(&mut h, old_sig.clone(), &account, *recipient.address());
+        let timer = Instant::now();
+        let output = h.run_raw(transaction);
+        let exe_time = timer.elapsed();
+        all_times.push(exe_time);
+        assert_success!(output.status().clone());
+    }
+
+    all_times.sort();
+    println!("P50={:?}", all_times[50]);
```

Comment:
Are you running this with `cargo test --release` I hope? :)

Reply:
yes